openpdf/src/main/java/com/lowagie/text/pdf/PdfOutline.java

```diff
@@ -268,9 +268,9 @@ public PdfOutline(PdfOutline parent, PdfAction action, Paragraph title) {
      */
     public PdfOutline(PdfOutline parent, PdfAction action, Paragraph title, boolean open) {
         super();
-        StringBuffer buf = new StringBuffer();
-        for (Iterator i = title.getChunks().iterator(); i.hasNext(); ) {
-            Chunk chunk = (Chunk) i.next();
+        StringBuilder buf = new StringBuilder();
+        for (Object o : title.getChunks()) {
```

Comment:
The next iteration of changes should be to add proper return types to things like `Phrase.getChunks()`, removing the need for this cast.